src/message/unlock-connector/index.tsx

```diff
@@ -54,7 +54,7 @@ export class UnlockConnectorRequest {
 
   @IsNumber()
   @IsNotEmpty()
-  connectorId!: number;
+  connectorId?: number;
```

Comment:
changing to `?` only affects the typescript but shouldn't affect the validation for the field right? 

also, isn't connectorId required, so in theory the UnlockConnectorRequest shouldn't ever have a null connectorId? 🤔

Reply:
ah you are right, I realized that this was already handled here, I reverted the change! Thank you 🙇